From 5bd3978da391d2ebb3698ab782300ca260ecf325 Mon Sep 17 00:00:00 2001
From: Raptor Engineering Development Team <support@raptorengineering.com>
Date: Mon, 30 Apr 2018 03:55:48 -0500
Subject: [PATCH 7/8] Disable fast reset unless explicitly enabled

Fast reset causes the OCC to randomly stop communicating with the BMC,
leading to a fan control failure and possible second CPU guard due
to the resultant FSI bus communication issues.

Disable until Issue 1699 (https://github.com/openbmc/openbmc/issues/1699) is fixed
---
 core/platform.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/platform.c b/core/platform.c
index f09ea3c1..5872d75b 100644
--- a/core/platform.c
+++ b/core/platform.c
@@ -60,8 +60,8 @@ static int64_t opal_cec_reboot(void)
 
 	console_complete_flush();
 
-	/* Try fast-reset unless explicitly disabled */
-	if (!nvram_query_eq("fast-reset","0"))
+	/* Don't try fast-reset unless explicitly enabled */
+	if (nvram_query_eq("fast-reset","1"))
 		fast_reboot();
 
 	if (platform.cec_reboot)
-- 
2.11.0

